sdk/eventgrid/azure-eventgrid/tests/test_eg_publisher_client.py

```diff
@@ -132,32 +135,27 @@ def test_send_cloud_event_data_dict(self, resource_group, eventgrid_topic, event
                 )
         client.send(cloud_event)
 
+    @pytest.mark.skip("https://github.com/Azure/azure-sdk-for-python/issues/16993")
     @CachedResourceGroupPreparer(name_prefix='eventgridtest')
     @CachedEventGridTopicPreparer(name_prefix='cloudeventgridtest')
-    def test_send_cloud_event_data_base64_using_data(self, resource_group, eventgrid_topic, eventgrid_topic_primary_key, eventgrid_topic_endpoint):
+    def test_send_cloud_event_data_NULL(self, resource_group, eventgrid_topic, eventgrid_topic_primary_key, eventgrid_topic_endpoint):
```

Comment:
this test technically "passes", but with a NULL object sent in the request - skipping it for now

Reply:
You should be able to use "raw_request_hook" to inspect the body and confirm null was sent

Reply:
`AssertionError: assert '<azure.core.serialization._Null object at 0x048B9670>' is None`